### PR TITLE
give parameter type

### LINKED ISSUE #981: Compiling lighthouse-cli causes typescript error
**summary**
On the latest build, compiling the lighthouse-cli directory results in a compile error. 
screen shot 2016-11-19 at 6 13 33 pm

**steps to reproduce**
1. Clone lighthouse
2. [Follow build steps](https://github.com/GoogleChrome/lighthouse#develop)

Source of error [here](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-cli/chrome-launcher.ts#L167).

**Fix**
Add a type to parameter.
PR [here](https://github.com/GoogleChrome/lighthouse/pull/982)

### PR DESCRIPTION
**Pull request by olingern**
Fixes #981

### CODE REVIEW

**Comment by samccone** [comment]
it is concerning that this passed on CI in the first place. :( 

Thanks for the fix!

File: lighthouse-cli/chrome-launcher.ts [hunk 1]
```diff
@@ -164,14 +164,14 @@ class ChromeLauncher {
...
-      client.once('connect', _ => {
+      client.once('connect', (_: any) => {
         this.cleanup(client);
         resolve();
       });
     });
   }
 
-  // resolves when debugger is ready, rejects after 10 polls
+  // resolves when debugger is ready`, rejects after 10 polls
```
Comment (samccone) [inline lighthouse-cli/chrome-launcher.ts]: looks like a typo

File: lighthouse-cli/chrome-launcher.ts [hunk 2]
```diff
@@ -164,14 +164,14 @@ class ChromeLauncher {
         this.cleanup(client);
         reject(err);
       });
-      client.once('connect', _ => {
+      client.once('connect', (_: any) => {
```
Comment (samccone) [inline lighthouse-cli/chrome-launcher.ts]: instead of this _ hack to omit () we should just use `() => {` as there are no arguments to this fn.

**Comment by olingern** [comment]
> it is concerning that this passed on CI in the first place. :(
Indeed :|

No problem, and requested changes made!

**Pull request merged**